### DISCUSSION TITLE
Seeking Community advise: make mathematical Equations more accessible

### DISCUSSION THREAD

**Discussion by Adriani90** [nvaccess/nvda#14781] [Ideas]
### Is your feature request related to a problem? Please describe.
A long while ago Math ML support has been introduced to NVDA. however, there are still alot of publications and practices that do not use Math ML per se but create equations with Unicode alphanumeric mathematical symbols (see for example the equations feature in Microsoft Word).
Issues #4631 and #11526 describe the requests in more detail.
Another down side as of now is that we rely on external parties when handling math symbols, be it the Math ML software or the vendors of the synthesizer. Bugs in these third parties solutions pose a significant risk and people are often not able to continue studying or working on a professional level when such situations appear. There are different issues in NVDA repository dealing with math specific content.

### Describe the solution you'd like
We should try to minimize such dependencies on third party solutions as much as possible in this case, because there are already convenstions out there which are well implemented in the Unicode data base.
So I propose to include scientific typographie conventions in NVDA, of course there are limitations but we should try as far as possible.

### Practical
I have created a table containing all alphanumeric math and scientific symbols, and I would like to prepare a pull request to integrate these in the symbols file. Having these symbols according to my tests, NVDA reports a lot of equations in Microsoft Word as well as in Browsers regardless of the synthesizer used.

### To be considered
* In column c is the expected level one NVDA reporting, that would be the reporting when the user navigates with the focus to the character
* In column j we see the Level 4 of NVDA reporting, this would include all details of a symbol such as mathematical style, general font style, and the font itself.
* The words "small" and "digit" are excluded from the reporting completely since these are not important for the user, they are deductible because the digit is reported and the capital letter is reported with "capital"
* The reporting of "Capital" is actually redundant since it is already controlled by the pitch of the synthesizer, even for the unknown unicode characters.

### Steps of implementation
1. Adding the symbols with the level 1 NVDA Reporting according to this table to the symbols.dic (I can prepare a PR)
2. Adding the additional information included in level 4 NVDA reporting according to this table to the reporting of formating (NVDA+f / NVDA+shift+f); cc: @lukaszgo1 
3. Include the formating information in the reporting of formating properties when enabling the specific checkboxes in document formating settings (might improve efficiency in some cases when reading the equation)

### Expected outcome
NVDA would report the correct characters in an equation like F(f(x)) = cos(2x^2); 𝙵(𝚏(𝘹)) = cos(𝟤𝘹^𝟤) even if it is designed with alphanumeric mathematical unicode characters. No third party software required, regardless of the synthesizer used.
The style of the letters and digits (double-struck, fraktur, script, bold, italic etc.) are very important to understand the context of the equation, the user could retrieve the information by pressing NVDA+f or NVDA+shift+f or by turning on reporting of font and style in document formating settings.

### Describe alternatives you've considered
Alternative 1:
* Instead of reporting formating style on demand, we could use international conventions to report "variable" for italic, "vector or matrix" for bold and bold + italic, "Function" or "operator" for mathematical roman or greek letters and digits. See this link for more information: https://iupac.org/wp-content/uploads/2016/01/ICTNS-On-the-use-of-italic-and-roman-fonts-for-symbols-in-scientific-text.pdf
* Given the contradictory usage of these conventions in different scientific areas, this alternative is not really feasible.
Alternative 2: leave as is, but people interested in professional mathematical or scientific work will need to do alot of workarounds to meaningfully contribute to these areas.

### Additional context
The greek alphabeth has other unicode and hexadecimal values, so the letters would not interfere with the scientific symbols.

Would NV Access accept some work on this?
cc: @michaelDCurran, @Qchristensen, @seanbudd 
[alphanumeric Mathematical symbols.xlsx](https://github.com/nvaccess/nvda/files/11126962/alphanumeric.Mathematical.symbols.xlsx)

**Comment by Adriani90** [comment]
You might have to use the review cursor to identify the capital letters in the reporting columns, or f2 to edit the cell in the b column  because I used formulas to exclude noisy reporting from other columns which are hidden.

**Comment by seanbudd** [comment]
NV Access would like to integrate [MathCAT](https://nsoiffer.github.io/MathCAT/) into NVDA core when possible.

A research presentation at CSUN from Indiana University suggested that NVDA + MathCAT works generally very well and leads in mathematics support for screen readers.
https://accessinghigherground.org/what-grade-did-your-screen-reader-get-in-arithmetic/

**Comment by Adriani90** [comment]
That's great to read and hear, this would definitely improve things with regards to Math ML. However, there are still tones of forums, discussion grups, publications and other public documents which do not use Math ML standard, so NVDA should at least provide the basic unicode support for math unicode characters.

**Comment by Adriani90** [comment]
I am thinking to adjust the number sets or power sets that we already have in the symbols.dic file to just report the alphanumeric signs as such without any interpretation or any formating.
Regarding the interpretation:
* a Double-struck letter can be used in different science areas and must not explicitely mean that the letter expresses a number set.
For example, double-struck e might mean an expected value in a probability or statistics context, an euclaidian space, but it is also used for electric field in physics, at least in some countries.
See
https://mathworld.wolfram.com/Doublestruck.html

https://math.stackexchange.com/questions/304030/what-does-the-double-lined-capital-mathbbe-not-the-sigma-stand-for

and

http://unicode.org/mail-arch/unicode-ml/y2002-m03/0571.html

Ideally NVDA reports just the letter e, and the user can retrieve the formating "double-struck" by demand when pressing nvda+f or nvda+shift+f. This would make equation reading much more fluent.

Often people who are reading relevant publications would most probably understand the context and could interpret by themselves what E and double-struck should mean.

**Comment by Adriani90** [comment]
An alternative approach to the NVDA+f command is to have the complete unicode name of the character spoken when pressing nvda+dot multiple times. We already report the HTML hex and decimal  entity on demand, so the unicode name would make this more comfortable and I think it requires less effort to implement.
